cdap-app-templates/cdap-etl/cdap-etl-lib/widgets/AmazonSQS.json

```diff
@@ -0,0 +1,41 @@
+{
+  "metadata": {
+    "spec-version": "1.0",
+    "artifact-version": "3.3.0-SNAPSHOT",
```

Comment:
artifact version, plugin name, and plugin type should not be in the metadata. 

Artifact version is redundant, you will be getting properties from a specific artifact already, which requires that you know the version

Type and name don't make sense here, as you will need to know the type and name already in order to look this up. Instead, the property name (filename) should contain the plugin type and name. Plugins with the same name but different types can and do have different config settings.